docs/pages-for-subheaders/rke1-hardening-guide.md

```diff
@@ -293,7 +295,6 @@ services:
   kubelet:
     extra_args:
       feature-gates: RotateKubeletServerCertificate=true
-      protect-kernel-defaults: true
```

Comment:
Same as before:

> This needs to be inside a specific tab depending on the CIS benchmark version, because it's still needed in CIS 1.23 and CIS 1.24.

Reply:
This flag was removed, but it still need to be kept because it's inside the v1.24 tab, no?